_docs/services/relational-database.md

```diff
@@ -21,6 +21,7 @@ Plan Name                | Description
 `medium-psql-redundant`  | Dedicated redundant medium RDS PostgreSQL DB instance                        | AWS RDS Latest   |
 `large-psql`             | Dedicated large RDS PostgreSQL DB instance                                   | AWS RDS Latest   |
 `large-psql-redundant`   | Dedicated redundant large RDS PostgreSQL DB instance                         | AWS RDS Latest   |
+`xlarge-psql-redundant`  | Dedicated redundant xlarge RDS PostgreSQL DB instance                        | AWS RDS Latest   |
```

Comment:
Thank you for this!  Sure enough, there are a small handful of others missing as well related to this.  We'll get the rest in place in short order. :-)